docs/writing-rules/testing-rules.md

```diff
@@ -31,6 +31,27 @@ Other than annotations there are three things to remember when creating tests:
 The `.test.yaml` file extension can also be used for test files. This is necessary when testing YAML language rules.
 :::
 
+## Testing autofix
+
+Semgrep's testing mechanism also provides a way to test the behaviour of any fixes defined in the rules.
+To define a test for autofix behaviour: 
+
+1. Create a new code file with the `.fixed` suffix before the annotation. 
+   For example, the autofix test file for a rule with test code in `path/to/rule.py` will be named `path/to/rule.fixed.py`.
+2. This file should contain the expected result of applying autofixes to the test code file.
```

Comment:
```suggestion
2. Within the autofix test file, enter the expected result of applying autofixes to the test code file.
```
Just writing it as a step/verb rather than a state.

Reply:
@s-santillan Just a nit, I would simplify the second half of the sentence also: `enter the expected result of applied autofix rule to the test code`.